File: Domovoy/Models/Services/ApartmentService.cs
namespace Domovoy.Models;

public class ApartmentService: ServiceBase
{
    public List<ApartmentHouse> ScopeHouses { get; set; }
    public ApartmentServiceType Type { get; set; }
}

public enum ApartmentServiceType
{
    Animals,
    Cleaning
}

File: Domovoy/ClientApp/src/api/models/ApartmentService.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ApartmentHouse } from './ApartmentHouse';
import type { ApartmentServiceType } from './ApartmentServiceType';
import type { ApplicationUser } from './ApplicationUser';

export type ApartmentService = {
    scopeHouses?: Array<ApartmentHouse> | null;
    type?: ApartmentServiceType;
    id?: number;
    name?: string | null;
    serivceProvider?: ApplicationUser;
};